Gemfile

```diff
@@ -1,2 +1,5 @@
 source 'https://rubygems.org'
-gem 'github-pages'
\ No newline at end of file
+gem 'github-pages'
+# Uncomment the following line for local development
+# See: https://github.com/envygeeks/jekyll-docker/issues/340
+# gem "webrick"
```

Comment:
Nitpick: I would change the order of the comments. Like this it isn't as obvious that you should uncomment the gem line

Reply:
Thanks. Will change that. :)

Reply:
Another nitpick: Also I just noticed that we use double quotes instead of single quotes here which is different from above